.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Fixes #ISSUE.
+
+
+## Pre-merge checklist
+
+<!--- A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
+
+- [ ] Docs
+   - `docs/` have been added/updated **OR**
+   - No updates to the docs necessary after careful consideration.
+
+- [ ] QC
+   - `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
+   -  No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
+
```

Comment:
```suggestion

- [ ] Account for any new packages
  - Open an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) to request addition. Can also open a PR. Python package changes should occur in [[requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full)](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full), and non-Python packages in [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile).
```

I'm not sure on the exact wording here, or whether it's better to open an issue or PR, or leave it open for both options.